x-pack/metricbeat/modules.d/aws.yml.disabled

```diff
@@ -15,6 +15,8 @@
   period: 5m
   metricsets:
     - cloudwatch
+  include_linked_accounts: true
+  owning_account: 111111111111
```

Comment:
```suggestion
  # owning_account: 111111111111
```
I would comment out this line just to make aws.yml easy to use (can be used directly without changing any config).

Reply:
Yup, that's a good point :)